q2_katharoseq/plugin_setup.py

```diff
@@ -60,3 +69,44 @@
                 'positive signal in samples with as few as 50 to 500 cells.',
     citations=[citations['minich2018']]
 )
+
+
+plugin.methods.register_function(
+    function=estimating_biomass,
+    inputs={},
+    parameters={'total_reads': MetadataColumn[Numeric],
+                'control_cell_extraction': MetadataColumn[Numeric],
+                'positive_control_column': MetadataColumn[Categorical],
+                'positive_control_value': Str,
+                'extraction_mass_g': MetadataColumn[Categorical],
+                'min_total_reads': Int,
+                'pcr_template_vol': Int,
+                'dna_extract_vol': Int},
+    outputs=[('estimated_biomass', EstimatedBiomass)],
+    input_descriptions={},
+    parameter_descriptions={
+        'total_reads': 'The total reads present in each sample.',
```

Comment:
'The total sum of the reads or ASVs for each sample. This is post-processing i.e. deblur. This is not rarified reads or anything like that - it will be the raw original count'